simulation-ros/src/phantomx_pincher_arm/phantomx_pincher_arm_moveit/scripts/pick_place.py

```diff
@@ -186,12 +237,14 @@ def ef_pose(target, arm, attemps = 10):
     robot.pincher_gripper.execute(gplan)
 
     # -----------------------------
-    #  Go to some pose
+    #  Vai para algum lugar
```

Comment:
all right, are you challenging us to pick up some Portugese?

Reply:
This is note supposed to be here. This is a strong signal that this branch came from ros-simulation...

This thing cannot be merged.

Reply:
What kind of behaviour? Maybe if you describe we can figure it out from the diffs.

Reply:
it is showing some errors in the vrep gui. It seem it is related to the bumpers in the turtlebot2i.lua script. I think it was caused by some modifications coming from the EnyuSafety branch.